src/app/pages/app-bar/contextual/contextual.component.scss

```diff
@@ -0,0 +1,105 @@
+@import '~@pxblue/colors/palette.scss';
+
+.toolbar-container {
+    position: sticky;
+    z-index: 1000;
+    top: 0;
+    overflow-x: hidden;
+}
+.selected-header {
+    z-index: -1;
+    position: absolute;
+    top: 0;
+    opacity: 0;
+    background: map-get($pxb-black, 500);
+    color: map-get($pxb-white, 50);
+    transition: opacity 200ms ease-in-out;
+    &.active {
+        opacity: 1;
+        z-index: 1;
+    }
+    .mat-icon {
+        color: map-get($pxb-white, 50);
+    }
+}
+
+.contextual-table-body {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    .contextual-table-container {
+        overflow: auto;
+        max-width: 800px;
+        width: 100%;
+        padding: 16px;
+        &.small {
+            padding: 0;
+            max-width: unset;
+        }
+    }
+    .delete-items-row {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        margin-bottom: 16px;
+    }
+    .desktop-delete-icon {
+        margin-right: 4px;
+        margin-bottom: 2px;
+        font-size: 16px;
+        height: 16px;
+        width: 16px;
+    }
+    .contextual-table {
+        width: 100%;
+        .selected {
+            background-color: rgba(map-get($pxb-blue, 500), 0.05);
```

Comment:
It appears that the background opacity is overlapping. May want to do something like 

```suggestion
            background-color: rgba(map-get($pxb-blue, 500), 0.05), map-get($pxb-white, 50);
```

![image](https://user-images.githubusercontent.com/8997218/133670141-f8be6f03-a6b8-40f4-b3bb-1ac273836138.png)

Reply:
The sticky cell was overlapping each table's row, so we got a double background.  This should be fixed now.